apps/web/src/pages/templates/filter/FilterModal.tsx

```diff
@@ -312,22 +317,25 @@ function EqualityForm({
           }}
         />
       </Grid.Col>
+
       <Grid.Col span={spaSize}>
-        <Controller
-          control={control}
-          name={`steps.${stepIndex}.filters.0.children.${index}.value`}
-          defaultValue=""
-          render={({ field, fieldState }) => {
-            return (
-              <Input
-                {...field}
-                error={fieldState.error?.message}
-                placeholder="Value"
-                data-test-id="filter-value-input"
-              />
-            );
-          }}
-        />
+        {operator !== 'IS_DEFINED' && (
+          <Controller
+            control={control}
+            name={`steps.${stepIndex}.filters.0.children.${index}.value`}
+            defaultValue=""
+            render={({ field, fieldState }) => {
+              return (
+                <Input
+                  {...field}
+                  error={fieldState.error?.message}
+                  placeholder="Value"
+                  data-test-id="filter-value-input"
+                />
```

Comment:
@jainpawan21 Like we discussed previously, how is the value field is being handled if it contained a different value before selecting the IS_DEFINED operator.

Reply:
Added code to make it ""